pyproject.toml

```diff
@@ -26,11 +26,12 @@ dependencies = [
     "aiohttp",  # TODO: remove in favor of httpx
     "anyio",
     "coredis",
+    "fh-llm-client",
```

Comment:
I think we can remove `coredis` and `limits` too, and don't forget `mypy` in `pyproject.toml`/`.pre-commit-config.yaml`